static/scss/top-app-bar.scss

```diff
@@ -18,8 +19,9 @@
 
 .link-section.nowrap.login {
   ul {
+    margin: 0;
```

Comment:
This isn't being applied in non-mobile widths. I think instead of using this `.link-section.nowrap.login ul`, you should apply it directly to `.navbar-collapse` (which is one of the classes applied to this `ul`), or any other selector that will apply `margin: 0` to this `ul` at all widths